Python/initconfig.c

```diff
@@ -2216,12 +2236,36 @@ config_usage(int error, const wchar_t* program)
         fputs(usage_1, f);
         fputs(usage_2, f);
         fputs(usage_3, f);
-        fprintf(f, usage_4, (wint_t)DELIM);
-        fprintf(f, usage_5, (wint_t)DELIM, PYTHONHOMEHELP);
-        fputs(usage_6, f);
+        fputs(usage_4, f);
     }
 }
 
+static void
+config_envvars_usage()
+{
+    printf(usage_envvars1, (wint_t)DELIM);
+    printf(usage_envvars2, (wint_t)DELIM, PYTHONHOMEHELP);
+    puts(usage_envvars3);
+    puts(usage_envvars4);
+    puts(usage_envvars5);
+}
+
+static void
+config_xoptions_usage()
+{
+    puts(usage_xoptions);
+}
+
+static void
+config_complete_usage(const wchar_t* program)
+{
+   config_usage(0, program);
+   puts("\n");
```

Comment:
Will `putchar` be better for a single letter output? I mean, semantically, not for individual bytes and nanoseconds.

```suggestion
   putchar('\n');
```

Reply:
The intent here is «put things between sections», not «put one char specifically», so no :)